src/relay/transforms/gradient.cc

```diff
@@ -106,14 +106,67 @@ struct ADValueNode {
   }
 };
 
+Expr MultiZerosType(const Type& t) {
```

Comment:
can you unify MultiZeros with MultiOnes?

Reply:
I just pushed that. Thank you for the suggestion.